proposals/0220-count-where.md

```diff
@@ -3,7 +3,7 @@
 * Proposal: [SE-0220](0220-count-where.md)
 * Author: [Soroush Khanlou](https://github.com/khanlou)
 * Review Manager: [Chris Lattner](https://github.com/lattner)
-* Status: **Accepted**
+* Status: **Implemented**
```

Comment:
SE-0220 was reverted (in apple/swift#22289 and apple/swift#22382) so please leave this as accepted.